pllug-presentation-system/presentation-data/documentimportstrategy.h

```diff
@@ -0,0 +1,15 @@
+#ifndef DOCUMENTIMPORTSTRATEGY_H
+#define DOCUMENTIMPORTSTRATEGY_H
+
+#include "presentation.h"
```

Comment:
It is better to use forward declaration here instead of include "presentation.h", as we have a pointer to Presentation.